### DISCUSSION TITLE
What is area?

### DISCUSSION THREAD

**Discussion by ScottWOlson** [weather-gov/api#342] [Q&A]
Have tried to make request for satellite thumbnail but with no luck. For area I've tried two letter state, forecast office, obs and lat lon but just get an error. Can you share an example of a proper URL to get an image?

**Comment by scadergit** [comment]
Hi, please provide some examples of the links and the error messages that were returned.

**Comment by ScottWOlson** [comment]
https://api.weather.gov/thumbnails/satellite/FSD

https://api.weather.gov/thumbnails/satellite/KBKX

https://api.weather.gov/thumbnails/satellite/CA

https://api.weather.gov/thumbnails/satellite/36.77,-119.41


{
  "correlationId": "a9775b32-8b57-45dc-8c7e-e15667cb5db8",
  "title": "Unexpected Problem",
  "type": "https://api.weather.gov/problems/UnexpectedProblem",
  "status": 500,
  "detail": "An unexpected problem has occurred.",
  "instance": "https://api.weather.gov/requests/a9775b32-8b57-45dc-8c7e-e15667cb5db8"}

Response headers

 cache-control: private, must-revalidate, max-age=2592000
content-length: 325  content-type: application/problem+json  expires:
Thu, 18 Jun 2020 13:30:03 GMT  pragma: no-cache

**Comment by StephenClouse** [comment]
Personally I would advise simply not using this endpoint. It's almost certainly not what you think it is and will be deprecated in an upcoming release.

The (very small) images it provides are static and do not update. If you are looking for current satellite imagery, you probably want to look at [NESDIS](https://www.star.nesdis.noaa.gov/GOES/index.php).

**Comment by ScottWOlson** [comment]
Okay I got you, yes I was actually interested in the thumbnail for a
weather app,even static.

A couple things, is there any way to get an observation for a lat/lon,
without having to first call to get all the endpoints?

Also, structurally, it's a bit hard to work with and much different than
all the other weather APIs we've worked with. For instance wind separately
would be in degrees as well as wind speed. Some fields are missing too.

**Comment by scadergit** [comment]
Given lat/lon permutations hinder necessary cache optimizations, only the /points endpoint will accept those parameters. Furture efforts will revisit this approach, but right now performance of the service is the primary focus.

The forecast data needs some improvement, but it has not been given priority yet for similar reasons.

Sorry for the delayed reply.
